secret.txt

```diff
@@ -1 +1 @@
-Mon animal préféré est le caribou
+Le meilleur animal de tous les temps est le caribou
```

Comment:
Je refuse cette pull request, le meilleur animal c'est le phasme.

Sources : https://genius.com/discussions/41430-Top-10-best-animals-of-inside-the-entire-world